### ISSUE TITLE
Markdown doesn't interpret backticks properly on Teams when using an AdaptiveCard

### ISSUE THREAD

**Issue by nmarkey10** [microsoft/AdaptiveCards#4906]
# Platform

What platform is your issue or question related to? (Delete other platforms).

* JavaScript/Typescript

# Author or host

Currently creating cards to send to Teams via TypeScript express -> express-msteams-host -> botbuilder (cardfactory) -> back to teams.  The backtick to create preformatted text does not currently work.

# Issue

Issue is stated above.  Any help is appreciated!

**Comment by sparrowt** [comment]
Yes sadly backticks for `preformatted` or code snippets is not listed on the subset of markdown supported by Adaptive Cards [here](https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-format?tabs=adaptive-md%2Cdesktop%2Cconnector-html#format-cards-with-markdown) whereas if you switch to the 'connector cards' tab within that panel then it is there.

It's sad that this functionality which the legacy `MessageCard` supported has been removed from `AdaptiveCard` 😢

**Comment by fraghag** [comment]
The triple backticks work here:
https://adaptivecards.io/designer
but not in teams. Seems like a bug to me!

**Comment by atc0005** [comment]
See also:

- https://github.com/microsoft/AdaptiveCards/issues/8995
